gcloud/bigquery/dataset.py

```diff
@@ -13,13 +13,10 @@
 # limitations under the License.
 
 """Define API Datasets."""
-
-import datetime
-
-import pytz
 import six
 
 from gcloud.exceptions import NotFound
+from gcloud.bigquery._helpers import _datetime_from_prop
```

Comment:
Not a big deal but `gcloud.bigquery` comes before `gcloud.exceptions` in alphabetical order.